### ISSUE TITLE
Allow players to export material list into plain text file

### ISSUE THREAD

**Issue by kostanakis** [Lunatrius/Schematica#83]
It would be great to have such option, because it's quite convenient to print the list on paper and use it in other world with no need to keep the list in mind.

**Comment by Lunatrius** [comment]
Something like this (need to make it look "pretty")?
```
16 Acacia Door (minecraft:acacia_door)
16 Acacia Fence (minecraft:acacia_fence)
16 Acacia Fence Gate (minecraft:acacia_fence_gate)
4 Acacia Leaves (minecraft:leaves2)
2 Acacia Sapling (minecraft:sapling)
4 Acacia Wood (minecraft:log2)
1 Acacia Wood Planks (minecraft:planks)
4 Acacia Wood Slab (minecraft:wooden_slab)
16 Acacia Wood Stairs (minecraft:acacia_stairs)
12 Activator Rail (minecraft:activator_rail)
1 Allium (minecraft:red_flower)
```

**Comment by Lunatrius** [comment]
Implemented in 6e4ab3521c19fe9da0ee098e9fa9055e76d01171.
